lib/components/Field/FormField.spec.tsx

```diff
@@ -1,16 +1,17 @@
 import * as React from 'react';
-import * as sinon from 'sinon';
-import * as classNames from 'classnames/bind';
-import { expect, assert } from 'chai';
+import { expect } from 'chai';
 import { mount } from 'enzyme';
-import FormField, { FormFieldProps } from './FormField';
-import { keyCode } from '../../Common';
-import { TestHookWrapper } from '../../common/testHookWrapper.spec';
+import FormField from './FormField';
 import { describe, it } from 'mocha';
 
+const f1 = 112;

```

Comment:
get from keycode enum?

Reply:
i tried that and the test was failing.  i'll try again.